### PR TITLE
Update scss add animation and fix footer

### PR DESCRIPTION
**Pull request by remy-rm**

### CODE REVIEW

File: package.json
Comment (remy-rm) [inline package.json:1]: add library lucide (icon )

File: src/app/app.component.html
Comment (remy-rm) [inline src/app/app.component.html:1]: fix footer at the bottom

File: src/app/component/navbar/navbar.component.html
Comment (remy-rm) [inline src/app/component/navbar/navbar.component.html:1]: remove display none for future feature

File: src/app/component/recipes-display/recipes-display.component.html
Comment (remy-rm) [inline src/app/component/recipes-display/recipes-display.component.html:1]: change carrousel for infinite carrousel

File: src/app/component/recipes-display/recipes-display.component.scss
Comment (remy-rm) [inline src/app/component/recipes-display/recipes-display.component.scss:1]: add animation slide for infinite croussel

File: src/app/component/recipes-display/recipes-display.component.ts
Comment (remy-rm) [inline src/app/component/recipes-display/recipes-display.component.ts:1]: adapte logic infinite caroussel

File: src/app/pages/about/about.component.scss
Comment (remy-rm) [inline src/app/pages/about/about.component.scss:1]: adjust css for no scroll in this page

File: src/app/pages/homepage/homepage.component.scss
Comment (remy-rm) [inline src/app/pages/homepage/homepage.component.scss:1]: fix background img

File: src/app/pages/single-recipe/single-recipe.component.scss
Comment (remy-rm) [inline src/app/pages/single-recipe/single-recipe.component.scss:1]: css redesign for this page

**Review by Benja-san — APPROVED** [review]

**Pull request merged**